### PR TITLE
Added more platform event fields

### LINKED ISSUE #91: Add Platform Event fields for org, user, and record details
Currently, there are a handful of fields on `LogEntryEvent__e` related to the org, user and related record - but most of the detailed info is later added asynchronously on `Log__c` where other fields for the org, user, profile, role, etc are populated. This was done to help minimize Logger's CPU usage - but having some of these details on the platform event object would let external systems easily get it directly from `LogEntryEvent__e`. 

General approach: 
- Cache relevant queries in private static variables (can't assume the org will have platform cache) and add private methods to `LogEntryEventBuilder` for setting fields
- Update `LogEntryEventHandler` to simplify code - it should just map fields from the platform event object to the log object

### LINKED ISSUE #97: Current session unavailable - System.UnexpectedException
I'm trying to implement some logging in a scheduled job and got the following:

Any ideas what to check that could be causing this?

```
25915:20:00.0 (286295243)|METHOD_EXIT|[55]|01p3f000000TQ0F|Nebula.Logger.saveLog()
26015:20:00.0 (286460664)|FATAL_ERROR|System.UnexpectedException: Current session unavailable
261
262Class.Auth.SessionManagement.getCurrentSession: line 5, column 1
263Class.Nebula.LogEntryEventBuilder.getSessionMap: line 308, column 1
264Class.Nebula.LogEntryEventBuilder.setUserSessionDetails: line 258, column 1
265Class.Nebula.LogEntryEventBuilder.getLogEntryEvent: line 240, column 1
266Class.Nebula.Logger.saveLog: line 689, column 1
267Class.Nebula.Logger.saveLog: line 656, column 1
268Class.InstructorsEmailSender.execute: line 55, column 1
26915:20:00.0 (286694887)|CODE_UNIT_FINISHED| REMOVED
27015:20:00.0 (286723785)|EXECUTION_FINISHED
```

Sample:
```
global class InstructorsEmailSender implements Schedulable {
    global void execute(SchedulableContext SC) {
        Nebula.Logger.info('Running the InstructorsEmailSender Class');
        Nebula.Logger.saveLog();
    }
}

```

### PR DESCRIPTION
**Pull request by jongpie**
- Closes #91 by replicating several fields to `LogEntryEvent__e` that were previously only available on `Log__c` or `LogEntry__c`. This gives any platform event subscribers more info about the org and user.
- Fixes #97 by changing how user session details are retrieved + added 3 new fields in the process (listed below), and deprecated `LogEntryEvent__e.LoginDomain__c` + `Log__c.LoginDomain__c`
  1. `AuthSession.LoginHistory.Application`
  2. `AuthSession.LoginHistory.Platform`
  3. `AuthSession.LoginHistory.Browser`
- Updated field arrangement on `Log__c` page layout + flexipage

### CODE REVIEW

File: nebula-logger/main/log-management/objects/Log__c/fields/LoginDomain__c.field-meta.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>LoginDomain__c</fullName>
     <externalId>false</externalId>
-    <label>Login Domain</label>
+    <label>DEPRECATED: Login Domain</label>
```
Comment (jamessimone) [inline nebula-logger/main/log-management/objects/Log__c/fields/LoginDomain__c.field-meta.xml:5]: nice

File: nebula-logger/tests/log-management/classes/LogEntryEventHandler_Tests.cls [hunk 2]
```diff
@@ -155,24 +156,23 @@ private class LogEntryEventHandler_Tests {
...
 
         // Org fields
-        System.assertEquals(Url.getOrgDomainUrl().toExternalForm(), log.OrganizationDomainUrl__c);
-        System.assertEquals(orgEnvironmentType, log.OrganizationEnvironmentType__c);
-        System.assertEquals(org.Id, log.OrganizationId__c);
-        System.assertEquals(org.InstanceName, log.OrganizationInstanceName__c);
-        System.assertEquals(org.Name, log.OrganizationName__c);
-        System.assertEquals(org.NamespacePrefix, log.OrganizationNamespacePrefix__c);
-        System.assertEquals(org.OrganizationType, log.OrganizationType__c);
+        //System.assertEquals(Url.getOrgDomainUrl().toExternalForm(), log.OrganizationDomainUrl__c);
```
Comment (jamessimone) [inline nebula-logger/tests/log-management/classes/LogEntryEventHandler_Tests.cls:164]: did you want to keep these commented out?

Comment (jongpie) [reply nebula-logger/tests/log-management/classes/LogEntryEventHandler_Tests.cls]: Whoops, I've uncommented & finished implementing them

**Comment by jongpie** [comment]
While fixing the auth issues in #97, I noticed 3 new fields that would be helpful - login application, platform & browser. I've added these fields to `LogEntryEvent__e` and `Log__c`

image

**Pull request merged**